app/controllers/concerns/sortable.rb

```diff
@@ -11,7 +11,7 @@ def sortable_columns
   end
 
   def order_by
-    "#{sort_column} #{sort_direction}"
+    "#{sort_column} #{sort_direction} NULLS LAST"
```

Comment:
This looks like a possible SQL injection vulnerability. We should probably wrap this with some defensive checks.

Reply:
The sort column and direction are being restricted to certain fields:

```ruby
  def order_by
    "#{sort_column} #{sort_direction} NULLS LAST"
  end

  def sort_column
    return params[:column] if sortable_columns.include?(params[:column])
    return "name"          if sortable_columns.include?("name")
    "created_at"
  end

  def sort_direction
    %w[asc desc].include?(params[:direction]) ? params[:direction] : "asc"
  end
```

Reply:
Nice job. I should have done more research.